src/libcollections/btree/node.rs

```diff
@@ -389,6 +389,7 @@ impl<K, V> Node<K, V> {
 
     #[inline]
     pub fn as_slices_internal_mut<'b>(&'b mut self) -> MutNodeSlice<'b, K, V> {
+        // TODO: Bad: This relies on structure layout!
```

Comment:
I think this won't pass `make tidy` (due to the TODO). Could you remove this and perhaps it can be just handled later?